### PR TITLE
Added logic for returning invite token as response field

### PR DESCRIPTION
**Pull request by icecream78**
Added logic for returning invite url as part of response according to issue #65
Resulting code is little bit messy, because netlify's original code isn't suit to our logic=)

### CODE REVIEW

**Comment by icecream78** [comment]
@kiwicopple is PR still needed?

File: mailer/mailer.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 // Mailer defines the interface a mailer must implement.
 type Mailer interface {
 	Send(user *models.User, subject, body string, data map[string]interface{}) error
-	InviteMail(user *models.User, referrerURL string) error
+	InviteMail(user *models.User, referrerURL string, notSendMail bool) (string, error)
```
Comment (kiwicopple) [inline mailer/mailer.go:16]: @awalias you happy with the param name here? Could be a bit confusing `notSendMail`, but I don't know of a better approach given the structure of the invite/mailer system

Comment (icecream78) [reply mailer/mailer.go:16]: I know that it's ugly. On the other hand I can implement return invite url without breaking interface, but we will have duplicate logic for generating invite link and I think that it's more painfull then ugly interface.

Comment (icecream78) [reply mailer/mailer.go:16]: @awalias answer please

Comment (kiwicopple) [reply mailer/mailer.go:16]: Hey @icecream78 - we're just wrapping up Launch Week so the team has been a bit busy. We'll get to this soon .

**Pull request closed without merging**